release_process.md

```diff
@@ -0,0 +1,5 @@
+## Release Process ##
+To create a release there are a few steps you should follow:
+- Update the version number in `version.py` and push this to master. We use [Semantic Versioning](https://semver.org/).
+- Create a new tag with the new version number: `v<major.minor.patch>`.
```

Comment:
You need to push the tag too. Add the tag and push commands you need to perform.